scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/LegacyScanner.scala

```diff
@@ -287,35 +287,38 @@ class LegacyScanner(input: Input, dialect: Dialect) {
     if (inStringInterpolation) return getStringPart(multiLine = startsStringPart(sepRegions.tail))
     else if (inXmlLiteral) return fetchXmlPart()
 
+    @inline def getIdentRestCheckInterpolation() = {
+      getIdentRest()
+      if (ch == '"' && token == IDENTIFIER)
+        token = INTERPOLATIONID
+    }
+
     (ch: @switch) match {
       case ' ' | '\t' | CR | LF | FF =>
         token = WHITESPACE
         strVal = ch.toString
         nextChar()
       // nextToken()
       case 'A' | 'B' | 'C' | 'D' | 'E' | 'F' | 'G' | 'H' | 'I' | 'J' | 'K' | 'L' | 'M' | 'N' | 'O' |
-          'P' | 'Q' | 'R' | 'S' | 'T' | 'U' | 'V' | 'W' | 'X' | 'Y' | 'Z' | '$' | '_' | 'a' | 'b' |
+          'P' | 'Q' | 'R' | 'S' | 'T' | 'U' | 'V' | 'W' | 'X' | 'Y' | 'Z' | '_' | 'a' | 'b' | // '$'
```

Comment:
```suggestion
          'P' | 'Q' | 'R' | 'S' | 'T' | 'U' | 'V' | 'W' | 'X' | 'Y' | 'Z' | '_' | 'a' | 'b' |
```

Reply:
i can do this, of course. keep in mind that without a `//` comment, the formatter will actually include a couple of pattern matches from the next line and end up redoing formatting for this and next two lines as well.

Reply:
That's fine, it's clear what the change here is.